### ISSUE TITLE
Issue when file path contains non standard characters such as é

### ISSUE THREAD

**Issue by msevestre** [Open-Systems-Pharmacology/rClr#3]
```
> sim <- loadSimulation("C:/tests/9.0/R Generation/Sim_é.pkml")
Error in rClr::clrCall(simulationPersister, "LoadSimulation", filePath) : 
```

**Comment by Yuri05** [comment]
s. also https://stackoverflow.com/questions/53870917/unicode-characters-returning-from-r-net

**Comment by PavelBal** [comment]
This has been solved? I can load a simulation with an `é` in the path...

**Comment by msevestre** [comment]
nope. Should still be the same. I wonder if this is a version dependent thing then...

**Comment by PavelBal** [comment]
I might have found the solution for the issue with unicode (which would also solve the problem with units without the workaround implemented by @msevestre )

It appears that what we type in is not actually UTF8, or not always UTF8, don't ask:

```
# R knows what "s" is
> utf8ToInt(x = "s")
[1] 115
# But obviously doesn't what "µ" is
> utf8ToInt(x = "µ")
[1] NA
```

We can "ensure" that the string is UTF8 by calling `enc2utf8(string)`

```
> utf8ToInt(x = enc2utf8("µ"))
[1] 181
```

This also works for the "HasUnit"-Problem:

```
# Show all units of the parameter "liverVolume"
> liverVolume$allUnits
[1] "l"  "ml" "µl"
# The unit "ml" is found
> rClr::clrCallStatic(WITH_DIMENSION_EXTENSION, "HasUnit", liverVolume$ref, "ml")
[1] TRUE
# "µl" however not
> rClr::clrCallStatic(WITH_DIMENSION_EXTENSION, "HasUnit", liverVolume$ref, "µl")
[1] FALSE

# This works
> rClr::clrCallStatic(WITH_DIMENSION_EXTENSION, "HasUnit", liverVolume$ref, enc2utf8("µl"))
[1] TRUE
```

I guess it would also work for paths etc. So we just have to ensure that we enclose every string we send to .NET in enc2utf8().

**Comment by msevestre** [comment]
Aynone knows if we can do it in the rCLr package? IT feels like the caller should not have to deal with this?

**Comment by Yuri05** [comment]
no idea. 
Alternatively encapsulate all string in rClr-Calls which are made in ospsuite-r

**Comment by msevestre** [comment]
sure. But this is tedious and error prone (because we have to think about it all the time)

**Comment by PavelBal** [comment]
> Aynone knows if we can do it in the rCLr package? IT feels like the caller should not have to deal with this?

I think it should be possible, but now idea where.
